sass/styles.scss

```
@@ -0,0 +1,338 @@
+* {
+    box-sizing: border-box;
+}
+
+@mixin large-text {
+    font: {
+        family: Helvetica, sans-serif;
+        size: 8em;
+        weight: normal;
+    }
+
+    color: white;
+}
+
+@mixin medium-text {
+    font: {
+        family: Helvetica, sans-serif;
+        size: 1em;
+        weight: normal;
+    }
+}
+
+@mixin small-text {
+    font: {
+        family: Helvetica, sans-serif;
+        size: 0.8em;
+        weight: normal;
+    }
+}
+
+html,
+body,
+.wrapper {
+    height: 100%;
+}
+
+body {
+    margin: 0;
+}
+a {
+    color:black;
+}
+h1 {
+    @include large-text;
+    margin: 0;
+    padding: 25px 5px;
+}
+
+p {
+    @include medium-text;
+    margin: 0;
+}
+
+ul {
+    padding: 0;
+
+    li {
+        @include small-text;
+        list-style: none;
+    }
+}
+
+.wrapper {
+    margin: auto;
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    padding: 1em;
+}
+
+// upper
+.upper-box {
+    position: relative;
+    width: 100%;
+    padding-bottom: 50px;
+}
+
+.title-wrapper {
+    display: grid;
+    grid-template-columns: 33.33% 33.33% 33.33%;
```

Consider using repeat?
```suggestion
    grid-template-columns: repeat(3, 33.33%);
```